src/lib/terra-components.module.ts

```diff
@@ -17,6 +17,7 @@ import { DragDropModule } from '@angular/cdk/drag-drop';
 import { ModalModule } from 'ngx-bootstrap/modal';
 import { MatIconModule } from '@angular/material/icon';
 import { MatCheckboxModule } from '@angular/material/checkbox';
+import { TerraFilterModule } from './components/filter/filter.module';
```

Comment:
Das muss hier nicht rein. Wir bieten es ja separat an
```suggestion
```

Reply:
müsste ich rein nehmen weil Terra gemeckert hat

Reply:
Stattdessen musst du die public-api.ts anpassen

Reply:
Wieso hat Terra gemeckert?

Reply:
weil das Module dort verwendet wird!

Reply:
Ja aber das soll ja separat importierbar sein.
Deshalb muss es hier raus und stattdessen in der public-api.ts exportiert werden
Sodass man es in Terra dann wie folgt importieren kann:
`import { TerraFilterModule } from '@plentymarkets/terra-components';`

Reply:
ich teste das mal kurz, ob nach den finale Änderungen auch gemeckert wird

Reply:
Und wie war das Ergebnis?

Reply:
jo hat geklappt! hatte nur ein import in einem Module vergessen gehabt, deshalb hat es Terra nicht so gefallen :)

Reply:
Dann entferne es hier bitte.